src/CMakeLists.txt

```diff
@@ -1 +1,265 @@
-PROCESS_SOURCES()
+# CMakeLists.txt for libyui/src
+
+include( ../VERSION.cmake )
+include( GNUInstallDirs )       # set CMAKE_INSTALL_INCLUDEDIR, ..._LIBDIR
+
+find_package( Boost REQUIRED )
+
+
+set( TARGETLIB          libyui )
+set( TARGETLIB_BASE     yui    )
+
+set( HEADERS_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR}/yui )
+
+# if DESTDIR is set, CMAKE_INSTALL_INCLUDEDIR already contains it
+# during "make install" (but not for other make targets!):
+#
+#    sudo make install DESTDIR=/work/foo
+# or
+#    DESTDIR=/work/foo sudo make install
```

Comment:
sudo needs `-E` / `--preserve-env` for this to work so I'd rather omit the line entirely from the comment
```suggestion
#    DESTDIR=/work/foo sudo --preserve-env make install
```

Reply:
Right. Let's keep it simple rather than adding confusion. Removing the second version.